backend/django_backend/user/models.py

```diff
@@ -1,15 +1,19 @@
 from django.db import models
 from django.contrib.auth.models import User
 
-class Profile(models.Model):
+def upload_to(instance, filename):
+    return 'images/{filename}'.format(filename=filename)
+
+class Avatar(models.Model):
     user = models.OneToOneField(
         User,
         on_delete=models.CASCADE,
         primary_key=True,
     )
-    # avatar = models.ImageField(upload_to='profile_pics', default='default.jpg')
+    image = models.ImageField(upload_to=upload_to, blank=True, null=True)
```

Comment:
I think the default image is needed.
"Users can upload an avatar, with a default option if none is provided"

Reply:
When i was using the default.jpg, that was causing issues did not let update the image. So I changed to that way,  that means that the form doesn't require a value but the database does.

Reply:
Ok so we need to have some sort of front-end solution to show a default avatar if the avatar is not set?